code/XMLTester.xml

```diff
@@ -1457,10 +1457,64 @@
 
 <!--><-->
 
-<!-->CONF_DISCOVERY_SERVER<-->
+<!-->CONF_DISCOVERY_SERVER_MAIN_SCENARIO<-->
 
+<!-->CONF_DS_MAIN_SCENARIO_SERVER<-->
```

Comment:
The `discoveryServer_client_syncperiod` is specified in the C++, but not in the XML